app/js/sanitiseAdmin.js

```diff
@@ -0,0 +1,23 @@
+
+
+//add in isEmailValid from validateUserAdditions to validate edit email
+
+//to make sure name and email fields arent empty
```

Comment:
Refactor so that the doc blocs are consistent

Reply:
resolved